compose/utils.py

```diff
@@ -60,13 +62,21 @@ def split_buffer(stream, splitter=None, decoder=lambda a: a):
             yield item
 
     if buffered:
-        yield decoder(buffered)
+        try:
+            yield decoder(buffered)
+        except ValueError:
+            log.error(
+                'Compose tried parsing the following chunk as a JSON object, '
+                'but failed:\n%s' % repr(buffered)
+            )
+            raise
```

Comment:
Will this still result in a stack trace being printed? If so, it'd be good to handle the error such that that doesn't happen (perhaps by raising something that `main.py` can catch and handle).